### PR TITLE
[Android][Test][MBL-14664]: Removed all traces of okreplay/ditto support from our repo.

### PR DESCRIPTION
**Pull request by joehoag**
There is a separate ticket to remove the tapes from our vault repo.

### CODE REVIEW

File: automation/espresso/build.gradle [hunk 1]
```diff
@@ -183,33 +169,4 @@ dependencies {
...
-        licenses = ['Apache-2.0']
-        vcsUrl = 'https://github.com/instructure/mobile-shared.git'
-
-        version {
-            name = System.getenv("MVN_VERSION")
-            vcsTag = System.getenv("MVN_REVISION")
-            released = new Date().toString()
-        }
-    }
-}
```
Comment (joehoag) [inline automation/espresso/build.gradle:215]: Removing the bintray and publishing sections wasn't really ditto-related.  But they were no longer necessary.

**Comment by JordanMarshall** [comment]
I'm seeing these remaining references to Ditto (most of them are just comments):

**automation/canvas_espresso/build.gradle**
```
41:        def dittoMode = findProperty("dittoMode") ?: "play"
42:        buildConfigField "String", "GLOBAL_DITTO_MODE", "\"$dittoMode\""
43:        println "dittoMode: $dittoMode"
```

**automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt**
```
55: abstract class CanvasTest : InstructureTest(BuildConfig.GLOBAL_DITTO_MODE) {
```

**automation/dataseedingapi/src/main/kotlin/com/instructure/dataseeding/model/SubmissionApiModel.kt**
```
37: // OkReplay/Ditto don't like to traffic in Lists of things, so we made this extraneous class to
```

**automation/dataseedingapi/src/main/kotlin/com/instructure/dataseeding/model/AssignmentApiModel.kt**
```
76: // OkReplay/Ditto don't like to traffic in Lists of things, so we made this extraneous class to
```

**automation/espresso/src/main/kotlin/com/instructure/espresso/page/InstructureTest.kt**
```
36: abstract class InstructureTest(overrideDittoModeString: String? = null) : InstructureTestingContract {
```

**automation/espresso/src/main/kotlin/com/instructure/espresso/InstructureRunner.kt**
```
46:    // The caller can also use this opportunity to set up Ditto functionality on the client.
```

**apps/student/src/androidTest/java/com/instructure/student/ui/LoginSignInPageTest.kt**
```
25:    // No Ditto
```

**apps/student/src/androidTest/java/com/instructure/student/ui/LoginLandingPageTest.kt**
```
25:    // No Ditto
```

**apps/student/src/androidTest/java/com/instructure/student/ui/LoginFindSchoolPageTest.kt**
```
24:    // No Ditto
```

**apps/teacher/src/androidTest/java/com/instructure/teacher/ui/LoginSignInPageTest.kt**
```
27:    // Ditto doesn't support WebViews
```

**apps/teacher/src/androidTest/java/com/instructure/teacher/ui/LoginLandingPageTest.kt**
```
10:    // Ditto doesn't support WebViews
18:    // Ditto doesn't support WebViews
```

**apps/teacher/src/androidTest/java/com/instructure/teacher/ui/NotATeacherPageTest.kt**
```
25:    // Ditto doesn't support WebViews
```

**Comment by joehoag** [comment]
Thanks -- I'll address all of those.

**Comment by joehoag** [comment]
:-)

**Pull request merged**